test/fixtures/s3-fixture/docker-compose.yml

```diff
@@ -15,6 +15,21 @@ services:
     ports:
       - "80"
 
+  s3-fixture-other:
```

Comment:
Perhaps `s3-fixture-searchable-snapshots`?

Reply:
I prefer to not mention searchable snapshots in open source code, so I'll keep "other" if that's ok for you.